pytorch_lightning/loggers/mlflow.py

```diff
@@ -149,6 +151,15 @@ def experiment(self) -> MlflowClient:
             self.logger.experiment.some_mlflow_function()
 
         """
+
+        if self._run_id:
+            try:
+                self._mlflow_client.get_run(self._run_id)
+                return self._mlflow_client
+            except mlflow.exceptions.MlflowException:
```

Comment:
I don't know, as i have seen at the mlflow documentation the [mlflow.get_run function](https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.get_run) return an exception if the run doesn't exists. 
Is it a problem for the library to do it this way? I'm open to suggestions for changes if this way of doing it is a problem :)

Reply:
I also have added tests for this behaviour

Reply:
Done! test passing!

Reply:
sorry for the late reply. 

> Is it a problem for the library to do it this way? I'm open to suggestions for changes if this way of doing it is a problem :)

Not directly a problem but if it can be avoided (as you did now) it would be preferred, because `MlflowException` is very general and is raised for other exceptions unrelated to getting the run (I assume given the name).

Reply:
Okay, so, with the way that is now implemented, can we resolve this issue? :)